frontend/src/app/home/home.component.spec.ts

```diff
@@ -24,9 +25,12 @@ describe('HomeComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [HomeComponent, RestaurantComponent],
-      imports: [MatCardModule],
-      providers: [{provide: ComponentFixtureAutoDetect, useValue: true}]
+      declarations: [HomeComponent],
+      imports: [MatCardModule, MatSidenavModule, BrowserAnimationsModule],
```

Comment:
These imports could be moved to a module that is shared by production code and test code (e.g. `UiModule`).

Reply:
Good idea .